queries/julia/highlights.scm

```diff
@@ -226,8 +226,11 @@
 (prefixed_command_literal
   prefix: (identifier) @function.macro) @string.special
 
+((string_literal) @string.documentation
+  (function_definition))
+
```

Comment:
How? That'd need to apply `@string` to all strings, and then check if theres a subsequent function definition to see if `@string.documentation` should be applied

Reply:
Or maybe I misunderstood, do you mean move this capture right after the `@string` one?

Reply:
No, I think `@string_literal` is the docstring (as opposed to `@string`)?

Otherwise this is fine, but you'd need to add more constructs -- just replicate the markdown injections.

Reply:
Great, thanks